src/applications/personalization/profile/sass/profile-info-table.scss

```diff
@@ -1,7 +1,7 @@
 $corner-size: 4px;
 
 .profile-info-table {
-  h3 {
```

Comment:
This was half the problem. The table title used to always be an h3, but now it's usually an h2.